src/server/api/routes/cvs.router.js

```diff
@@ -123,46 +31,21 @@ router.post('/', (req, res) => {
  *    responses:
  *      200:
  *        description: CV was patched
- *      5XX:
+ *      400:
  *        description: Unexpected error.
  */
 router.patch('/:id', (req, res, next) => {
   cvsController
-    .editModule(req.params.id, req.body)
-    .then((result) => res.json(result))
-    .catch(next);
-});
-
-/**
- * @swagger
- * /cvs/{ID}:
- *  delete:
- *    summary: Delete a cv
- *    description:
- *      Will delete a cv with a given ID.
- *    produces: application/json
- *    parameters:
- *      - in: path
- *        name: ID
- *        description: ID of the cv to delete.
- *    responses:
- *      200:
- *        description:  CV deleted
- *      5XX:
- *        description: Unexpected error.
- */
-router.delete('/:id', (req, res) => {
-  cvsController
-    .deleteModule(req.params.id, req)
+    .editCv(req.params.id, req.body)
```

Comment:
You are not responding with anything here. You should respond with the updated cv